### PR TITLE
ADDED: DevOps Section To Concepts

### PR DESCRIPTION
**Pull request by Sommos**
Added a DevOps section to the Concepts folder, filled with content about DevOps, mainly targetting Azure. We can add additional platforms like GitLab at a later date from people who have worked with it.

### CODE REVIEW

**Review by JDSherbert — APPROVED** [review]
lgtm

File: docs/Concepts/DevOps.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: DevOps
+nav_order: 1
+parent: Concepts
+has_children: false
+---
+
+# {{ page.title }}
```
Comment (VerzatileDev) [inline docs/Concepts/DevOps.md:9]: There shouldn't be an Hastag  " # " before the call  Use 
` {{ page.title }}
====================== `
 instead unless the title differs from the Title of the page within the URL and nagivation.

**Review by VerzatileDev — COMMENTED** [review]
Lgtm, though take a look at the comment.

File: docs/Concepts/DevOps.md (hunk 1 above)
Comment (Sommos) [reply docs/Concepts/DevOps.md:9]: Just pushed the fix, but this has already been merged.

**Pull request merged**